docs/source/script/get-started-with-scripting.rst

```diff
@@ -22,22 +22,22 @@ You are likely to interface most with these High Fidelity APIs:
 +-------------------------------------------------------------------------+------------------------------------------------------------------+
 | API(s)                                                                  | Description                                                      |
 +=========================================================================+==================================================================+
-| `Entities <https://apidocs.highfidelity.com/Entities.html>`_            | Lets you manipulate the entities around you, as long             |
+| `Entities <https://apidocs.vircadia.dev/Entities.html>`_            | Lets you manipulate the entities around you, as long             |
```

Comment:
Does spacing need to be maintained such that the "|"s line up? Here and elsewhere in other changed tables.

Reply:
Yes, it does, good catch. I believe the tables will break if that is not corrected.